docs/contents.md

```
@@ -0,0 +1,60 @@
+  Validating client connection preambles.
+
+## Roadmaps
+
+- [Outbound messaging roadmap](asynchronous-outbound-messaging-roadmap.md)  
+  Task list for implementing asynchronous outbound messaging.
+- [Wireframe 1.0 roadmap](wireframe-1-0-detailed-development-roadmap.md)  
+  Detailed tasks leading to Wireframe 1.0.
+- [Project roadmap](roadmap.md)  
+  High-level development roadmap.
+- [1.0 philosophy](the-road-to-wireframe-1-0-feature-set-philosophy-and-capability-maturity.md)  
+  Philosophy and feature set for Wireframe 1.0.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)